### PR TITLE
Updates to Descriptions, no-follow patterns, and readme

### PR DESCRIPTION
**Pull request by matt598**
Sets the OG Description for link previews
Removes the no-follow attribute for pages we want google to show others more of
Adds to the readme

### CODE REVIEW

File: petition/index.html [hunk 1]
```diff
@@ -5,10 +5,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>tbfighters.link</title>
     <meta name="description" content="Redirection server for TBfighters projects.">
-    <meta name="robots" content="noindex, nofollow">
-    <meta name="googlebot" content="noindex, nofollow">
+    <meta name="robots" content="noindex">
+    <meta name="googlebot" content="noindex">
     <meta property="og:title" content="tbfighters.link">
-    <meta property="og:description" content="Redirection server for TBfighters projects.">
+    <meta property="og:description" content="MSF Access Campaign">
```
Comment (YummyOreo) [inline petition/index.html:11]: could you change this to something like MSF Access Campaign's Petition to lower Danaher's tests costs

**Review by YummyOreo — COMMENTED** [review]
Also, it might be nice to change the title to be the name of the redirect. I.e. "Petition" or "P4A Memes"

File: petition/index.html (hunk 1 above)
Comment (matt598) [reply petition/index.html:11]: I was matching their og:title/description value.
image

Comment (YummyOreo) [reply petition/index.html:11]: ok, thanks for the information. Looks good then

**Review by YummyOreo — APPROVED** [review]

**Pull request merged**